src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1992,6 +1992,33 @@ def _disk_encryption_set_format(cmd, namespace, name):
 # endregion
 
 
+def process_image_version_create_namespace(cmd, namespace):
+    """
+    Validate the resource id from different sources
+    Only one of these arguments is allowed to provide
+    Check the format of resource id whether meets requirement
+    """
+    process_gallery_image_version_namespace(cmd, namespace)
+
+    input_num = (1 if namespace.managed_image else 0) + (1 if namespace.virtual_machine else 0) + \
+                (1 if namespace.image_version else 0)
+    if input_num > 1:
+        raise ArgumentUsageError(
```

Comment:
Please use the more specific error type `MutuallyExclusiveArgumentError` instead of `ArgumentUsageError`